### PR TITLE
Project Advisory Guideline 1st Draft

### PR DESCRIPTION
**Pull request by complexbrains**
Requires some areas to be discussed and filled. Here is the google doc associated with it https://docs.google.com/document/d/1rz67OZslES1-P76o74txQyRMpglb1PMwRR6G2LWWcoo/edit#heading=h.1wh9ovwv5vvn

### CODE REVIEW

**Comment by cmaumet** [comment]
Thanks @complexbrains! I've commented directly in your google doc. I'm pinging @anibalsolon as well for his input :D

File: .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Project Advisor Guideline
+
+Project Advisor (PA) takes actions in different stages of the project executions to make sure they fully follow the OHBM Hackathon procedures. Please do not forget that, as PA, you are not expected to be in expertise of the project you are assigned as PA, so please do not feel under pressure of being not familiar with the field. You will be there to guide them to use their time and effort wisely and help them to reach the right sources quickly as possible in accordance with the OHBM Brainhack Team. Your project leaders will also know that you are not obligated to give them any advice on the scope/concept of the project methodologically, technologically and theoretically but help in many other aspects it would help them to conduct the project.  Before you start make sure that
+
+- You understand the roles and responsibilities of the PA 
+- You have enough time to dedicate as it might be needed. 
+- Get familiar yourself with the Handbook, Guidelines and general flow of the Brainhack.
```
Comment (jhlegarreta) [inline .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md:7]: Adding the links to these documents could be useful.

File: .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Create your Mattermost account and enrol to the [Project Advisors Channel]() to convey further discussions. 
+- Share some details about yourself in the [PA Spreadsheet]().
+
+## Roles and Responsibilities
+
+Your roles from the stage of project submission to the end day of the Brainhack includes;
+
+- Reviewing project assigned with you or you have chosen
+- Conveying a continuous communication with the project leaders and members through the Brainhack to;
+- If you would like, share your thoughts/advice on the methodological, technological and theoretical aspects of the projects.
```
Comment (jhlegarreta) [inline .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md:17]: I'd remove `If you would like,`

File: .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  - Help them to find and contact with the experts from the community in the case of queries.
+  - Direct them to related Brainhack sources/tools to use/read.
+  - Check that there is no conflict or violation of CoC occurs among the project members.
```
Comment (jhlegarreta) [inline .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md:20]: I think this requires re-wording.

File: .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+1. Review the issue associated with the project and check whether the details listed in the checklist were provided or not.
+2. Contact with the project leader through the PR and declare that you are PA for this project and your role in the project through the 3 days of Brainhack as it is listed below. 
+3. If there is any missing information in the project submission issue, ask the project leader to provide the information. 
+4. Fill the PA sheet regarding the project/s you will be PA for.
+5. Fill the PA sheet regarding whether first initial contact with the project leader was made and you explained your role to them.
+6. Review the project start, teaming up and share your decision in the related area of the PA sheet. 
+
+
+### After the Project is Accepted
+1. Make the acceptance declaration to the project leader.
```
Comment (jhlegarreta) [inline .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md:34]: I think this requires re-wording.

File: .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+- If you would like, share your thoughts/advice on the methodological, technological and theoretical aspects of the projects.
+  - Help them to find and contact with the experts from the community in the case of queries.
+  - Direct them to related Brainhack sources/tools to use/read.
+  - Check that there is no conflict or violation of CoC occurs among the project members. 
+-  Run daily and brief project meetings to observe how the project is going, share your advice and suggestions, and check if there is any need of help regarding the project.
+
+### At the Beginning of the Project Submission
+
+1. Review the issue associated with the project and check whether the details listed in the checklist were provided or not.
+2. Contact with the project leader through the PR and declare that you are PA for this project and your role in the project through the 3 days of Brainhack as it is listed below.
```
Comment (jhlegarreta) [inline .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md:26]: Only place where a PR is mentioned. Is is a PR or an issue?

Comment (jhlegarreta) [reply .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md:26]: "(...) your role (...) as it is listed below." The role is listed above.

File: .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+2. Contact with the project leader through the PR and declare that you are PA for this project and your role in the project through the 3 days of Brainhack as it is listed below. 
+3. If there is any missing information in the project submission issue, ask the project leader to provide the information. 
+4. Fill the PA sheet regarding the project/s you will be PA for.
+5. Fill the PA sheet regarding whether first initial contact with the project leader was made and you explained your role to them.
+6. Review the project start, teaming up and share your decision in the related area of the PA sheet. 
+
+
+### After the Project is Accepted
+1. Make the acceptance declaration to the project leader.
+2. Make sure they have prepared a pitching presentation comply with the requirements of the Brianhack guidelines.
```
Comment (jhlegarreta) [inline .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md:35]: I think this requires re-wording.

Also, `Brianhack` -> `Brainhack`

File: .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+3. Make daily meetings with the project team members to discuss the progresses in the project and understand if there are any obstacles along to the way that requires OHBM Team’s or experts involvements. 
+4. Issue the progress of the projects in the spreadsheet. 
+
+
+### After the Brainhack Finishes
+
+1. Guide them towards the demo pitch that will be held at the end of the Brainhack.
+2. Mention to the project members the proceeding possibilities for the Brainhack projects that might lead them to a publication in the future.
```
Comment (jhlegarreta) [inline .github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md:43]: I've seen some discussions about this on mattermost, and AFAIK, there is no clear venue for the proceedings yet, so unless this is clear before the hackathon, I would remove it.

**Comment by cmaumet** [comment]
@complexbrains @anibalsolon: I've added a "role description" file for the project monitors that is a condensed version of what we had in the guidelines. This can be used to advertise the role (and then the guidelines provide a more in-depth view once people are enrolled).

As discussed elsewhere, I don't know if we should push this now as the number of submitted projects is still limited. Do you have thoughts? Thanks!

**Comment by cmaumet** [comment]
I am going to merge this now as we need to update the project checklist now that submission is officially open.

**Pull request merged**